week4-ts/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import Router from 'components/common/Router';
+import { ThemeProvider } from 'styled-components';
+import GlobalStyle from 'styles/globalStyle';
+
+import theme from './styles/theme';
+function App() {
+  return (
+    <>
+      <GlobalStyle />
+      <ThemeProvider theme={theme}>
+        <Router />
+      </ThemeProvider>
```

Comment:
```suggestion
      <ThemeProvider theme={theme}>
        <GlobalStyle />
        <Router />
      </ThemeProvider>
```

요렇게 하면 GlobalStyle 에서도 theme 을 사용할 수 있다는 거 !

Reply:
와하,,,,오하 대박 ㅠㅠㅠㅠ 내 궁금증이 해결데따 글로벌에서도 쓰고 싶으면 어쩌지 이러고 잇엇는데
ㅠㅠ흑